addons/web/static/src/views/graph/graph_renderer.js

```diff
@@ -16,6 +16,12 @@ import { cookie } from "@web/core/browser/cookie";
 
 const NO_DATA = _t("No data");
 
+// Measures for which value should be converted into HH:MM format
+export const HOURS_MEASURE_FIELDS = [
```

Comment:
Note: the formatter could also be used to format values displayed on the Y axis of the graph

Reply:
Hello @sami-odoo 

You should set `widget="float_time"` in the parent arch, and it will be overridden (with xpath) when hr_timesheet is installed, which is fine.

Reply:
so you will have to modify the xpaths in project_task_view_graph in order to get widget="timesheet_uom" instead of widget="float_time"